lib/src/bottom_sheets/cupertino_bottom_sheet.dart

```diff
@@ -4,10 +4,15 @@
 
 import 'dart:async';
 
-import 'package:flutter/cupertino.dart';
+import 'package:flutter/cupertino.dart' show CupertinoTheme;
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
-import 'package:flutter/material.dart';
+import 'package:flutter/material.dart'
```

Comment:
Why is `show` needed? There is no conflicts between both libraries no?

Reply:
That's right. 
My intention is to prevent further problems when using material widgets or parts of the material library without having checked if this could cause problems if the material is not in the WidgetTree.